### PR TITLE
story guide

### PR DESCRIPTION
**Pull request by chloeleichen**
Implement story guide: https://projects.invisionapp.com/share/TUSDL6GYN8V#/screens/367290948

Add a getting started button in story panel, and when clicked, play a video on how to make the best use of the story feature

### CODE REVIEW

File: lib/ReactViews/Story/StoryBuilder.jsx [hunk 1]
```diff
@@ -29,7 +29,8 @@ const StoryBuilder = createReactClass({
     return {
       editingMode: false,
       currentStory: undefined,
-      recaptureSuccessFul: undefined
+      recaptureSuccessFul: undefined,
```
Comment (soyarsauce) [inline lib/ReactViews/Story/StoryBuilder.jsx]: This should be `recaptureSuccessful`

File: lib/ReactViews/Story/story-builder.scss [hunk 2]
```diff
@@ -123,3 +123,42 @@
...
+  }
+}
+
+.VideoGuideWrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba($dark-with-overlay, 0.8);
```
Comment (soyarsauce) [inline lib/ReactViews/Story/story-builder.scss:186]: We should add a `color: transparent;` here too to get rid of the white border

**Review by soyarsauce — COMMENTED** [review]
Looks great @chloeleichen - did @philipgrimmett decide he didn't want the slide-up animation anymore?

**Comment by AnaBelgun** [comment]
This is very high priority

**Comment by soyarsauce** [comment]
phil: 'as close as you can to invision doc'

**Comment by AnaBelgun** [comment]
@soyarsauce to finish this please
@KeyboardSounds to review

**Comment by soyarsauce** [comment]
Hi @KeyboardSounds - new updates @ https://github.com/TerriaJS/terriajs/pull/3458/commits/60dacb914c95b6616b3163d69c0c4b46243a08db

**Review by KeyboardSounds — APPROVED** [review]

**Pull request merged**